src/Init/Data/List/BasicAux.lean

```diff
@@ -148,6 +148,15 @@ def getLastD : (as : List α) → (fallback : α) → α
   | [],   a₀ => a₀
   | a::as, _ => getLast (a::as) (fun h => List.noConfusion h)
 
+/--
+`O(n)`. Rotates the elements of `xs` to the left such that the element at
+`xs[i]` rotates to `xs[(i - n) % l.length]`.
+
+`n` defaults to `1`.
```

Comment:
Is it necessary to specify the default value of `n` given that the [docs will already include `(n : optParam Nat 1)`](https://leanprover-community.github.io/mathlib4_docs/Init/Data/List/BasicAux.html#List.rotateLeft)? 

![2024-04-15-18:18:22-screenshot](https://github.com/leanprover/lean4/assets/21346448/9c531861-843b-443d-88ed-6fa664c7cd0f)

Reply:
I would say no... The main places people see these doc strings are docgen and hovers, but the type signature including the optParam shows up in both places in this case.

Reply:
Sounds good. I removed the note about the default value in both rotate functions

Reply:
That makes sense - we can assume that readers are seeing the docstring in the context of the signature.